client/login/wp-login/login-links.jsx

```diff
@@ -7,9 +7,10 @@ import Gridicon from 'gridicons';
 import page from 'page';
 import PropTypes from 'prop-types';
 import React from 'react';
-import urlModule from 'url';
 import { connect } from 'react-redux';
+import { get } from 'lodash';
 import { localize } from 'i18n-calypso';
+import url from 'url';
```

Comment:
Any reason to import all of `url` instead of just `{ parse }`?

Reply:
👉 c829363